lynx/build.gradle

```diff
@@ -3,10 +3,10 @@ apply plugin: 'checkstyle'
 
 android {
   compileSdkVersion 24
-  buildToolsVersion "24.0.0"
+  buildToolsVersion "24.0.1"
```

Comment:
You'll have to update the travis configuration or keep the build config as is to be able to use continous integration.